src/app/pages/device-options/history/hum-graph/hum-graph.component.ts

```diff
@@ -0,0 +1,294 @@
+import {
```

Comment:
Ressemble beaucoup à composant existant

Reply:
Yeah, this component is very similar with the temp graph. I believe there is a better method which can share it with temp graph component. I will figure it out.